src/scala/Eq.scala

```diff
@@ -0,0 +1,17 @@
+package scala
+
+/** A marker class indicating that values of kind `T` can be compared. */
+class Eq[-L, -R]
```

Comment:
The comments for `Eq` and `eqAny` below need to be updated to mention `L` and `R` instead of `T`.